app/src/main/java/it/quip/android/adapter/SearchArrayAdapter.java

```diff
@@ -0,0 +1,72 @@
+package it.quip.android.adapter;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.TextView;
+
+import com.parse.ParseObject;
+
+import java.util.List;
+
+import it.quip.android.R;
+
+
+public abstract class SearchArrayAdapter <T extends ParseObject> extends ArrayAdapter<T> {
```

Comment:
Good use of generics. We might want to have a more involved getView for the different types but this is fantastic for now